mlnx_sai/src/sai_2740.xml

```diff
@@ -0,0 +1,243 @@
+<?xml version="1.0"?>
+<root>
+	<platform_info type="2700">
+
+		<!-- Device MAC address  -->
+		<device-mac-address>00:02:03:04:05:00</device-mac-address>
```

Comment:
the platform_info field isn't used currently
still, it should be correct value

this new 2740 file is the port mapping for the new system type 2740